packages/ui/src/FocusAreaCard.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/image'
+
+import { Heading } from '@filecoin-foundation/ui/Heading'
+import { Icon, type IconProps } from '@filecoin-foundation/ui/Icon'
+import type { StaticImageProps } from '@filecoin-foundation/utils/types/imageType'
+
+type FocusAreaImageProps = StaticImageProps & {
+  sizes?: string
+}
+
+export type FocusAreaCardProps = {
+  title: string
+  children: React.ReactNode
+  icon?: IconProps['component']
+  image?: FocusAreaImageProps
+}
+
+export function FocusAreaCard({
+  title,
+  children,
+  image,
+  icon,
+}: FocusAreaCardProps) {
+  return (
+    <li className="focus-area-card">
+      {image && <FocusAreaCard.Image image={image} />}
+      {icon && <FocusAreaCard.Icon icon={icon} />}
+      <div className="focus-area-card-content">
+        <Heading tag="h3" variant="lg">
+          {title}
+        </Heading>
+        {children}
+      </div>
+    </li>
+  )
+}
+
+FocusAreaCard.Image = function FocusAreaCardImage({
+  image,
+}: Pick<FocusAreaCardProps, 'image'>) {
+  if (!image) return null
+
+  return (
+    <Image src={image.data} alt={image.alt} className="focus-area-card-image" />
+  )
+}
+
+FocusAreaCard.Icon = function FocusAreaCardIcon({
+  icon,
+}: Pick<FocusAreaCardProps, 'icon'>) {
+  if (!icon) return null
+
+  return (
+    <div className="accent-icon-regular">
```

Comment:
maybe there's a better way to reference this class? ideally it should be `focus-area-card-icon`

Reply:
What about

```
  .accent-icon-regular {
    @apply theme('colors.icon.accent');
  }
```

and

```
.focus-area-card-icon {
   @apply theme('colors.icon.accent');
}
```

Since we do have

```
  --color-icon-primary: theme('colors.brand.primary.300');
  --color-icon-accent: theme('colors.brand.secondary.100');
  --color-icon-subtle: theme('colors.neutral.300'); /* CHECK */
  ```
  
  Happy to rename to `--color-icon-brand-primary` if that makes it clearer?
  
  What do you think?